tests/methods/df-casscf-1/input.dat

```diff
@@ -43,12 +43,12 @@ set forte{
   print                   0
 }
 Ecas_1 = energy('forte')
-compare_values(Ecas_1, psi4_casscf_1, 6, "FORTE CASSCF energy @ 1.5")
+compare_values(psi4_casscf_1, Ecas_1, 6, "FORTE CASSCF energy @ 1.5")
```

Comment:
What's the point of changing this?

Reply:
I think either I or Kevin put the order incorrectly before. The first argument is the reference value and the second is the computed value. Everything looks fine until the function throws the error and then the computed value actually shows the reference value, which is somewhat confusing.